test_runner/batch_pg_regress/test_neon_regress.py

```diff
@@ -20,19 +20,22 @@ def test_neon_regress(neon_simple_env: NeonEnv, test_output_dir: Path, pg_bin, c
     runpath = test_output_dir / 'regress'
     (runpath / 'testtablespace').mkdir(parents=True)
 
+    # Find the pg_regress binary and --bindir option to pass to it.
+    proc = pg_bin.run(['pg_config', '--libdir'], capture_output=True)
+    libdir = proc.stdout.decode().strip()
+    proc = pg_bin.run(['pg_config', '--bindir'], capture_output=True)
+    bindir = proc.stdout.decode().strip()
+    pg_regress = os.path.join(libdir, 'postgresql/pgxs/src/test/regress/pg_regress')
```

Comment:
IMO, that looks not even more magical, but have no better proposals.

Reply:
Agreed. We could have a shared fixture for calling `pg_regress`, that would at least avoid duplicating this in `test_neon_regress.py`, `test_pg_regress.py` and in `test_isolation_regress.py`.